src/core_plugins/metrics/public/components/vis_types/table/series.js

```diff
@@ -100,9 +100,14 @@ function TopNSeries(props) {
     >
       <div className="vis_editor__container">
         <div className="vis_editor__series-details">
-          <div onClick={props.toggleVisible}><i className={caretClassName}/></div>
+          <div>
+            <a onClick={props.toggleVisible}>
```

Comment:
This is an action and not a navigation. Could you use a [`button` instead of an `a`](https://github.com/elastic/kibana/blob/master/style_guides/accessibility_guide.md#use-button-and-a-href) for this, please, since an `a` without an `href` isn't different from a `div` for a screen reader and also semantics and stuff :-)